src/features/script-load.js

```diff
@@ -10,39 +10,40 @@ systemJSPrototype.register = function (deps, declare) {
   systemRegister.call(this, deps, declare);
 };
 
+systemJSPrototype.createScript = function (url) {
+  const script = document.createElement('script');
+  script.charset = 'utf-8';
```

Comment:
Would it save bytes to do `Object.assign(script, {charset: 'utf-8', ...})`

Reply:
No Object.assign in IE11 :)

Reply:
Uncaught SyntaxError: Unexpected token export when import ES6 module

es6.js file
export var demo = "hello";

Reply:
Hi @appsou, could you clarify what your comment means? Are you reporting a bug with a current version of systemjs? Or are you reporting a problem with this PR?

Not that systemjs does not support loading ES modules, as [explained here](https://github.com/systemjs/systemjs/blob/master/docs/module-types.md). It primarily supports loading `System.register` modules.